pkg/types/config.go

```diff
@@ -502,7 +502,7 @@ func (pl PartitionList) GetByNameOrLabel(name, label string) *Partition {
 
 type ElementalPartitions struct {
 	BIOS       *Partition
-	EFI        *Partition
+	Boot       *Partition `yaml:"bootloader,omitempty" mapstructure:"bootloader"`
```

Comment:
`bootloader` partition is named as `Boot` in the struct, just for the sake of short names.